src/scan/_util/isInRange.ts

```diff
@@ -0,0 +1,65 @@
+import { PixelRatio } from 'react-native';
+import { Point, Rect } from 'vision-camera-code-scanner';
+
+type TIsInRange = {
+  cornerPoints: Point[];
+  scanRegion: Rect;
+  screenHeight: number;
+  screenWidth: number;
+  videoHeight: number;
+  videoWidth: number;
+};
+
+export function isInRange({ cornerPoints, scanRegion, screenHeight, screenWidth, videoHeight, videoWidth }: TIsInRange) {
+  const pixelRatio = PixelRatio.get();
+
+  const actualScreenWidth = screenWidth * pixelRatio;
+  const actualScreenHeight = screenHeight * pixelRatio;
+
+  const multiplierX = videoWidth / actualScreenWidth;
+  const multiplierY = videoHeight / actualScreenHeight;
+
+  const top = scanRegion.top * pixelRatio * multiplierY;
```

Comment:
kunnen we deze berekeningen niet in de overlayhook steken?  die blijven eigenlijk altijd hetzelfde, deze util zou enkel moeten kijken of de cornerpoints hierin liggen 🤔